plugin-server/tests/shared/process-event.test.ts

```diff
@@ -1173,25 +1179,46 @@ test('snapshot event stored as session_recording_event', async () => {
         '',
         {
             event: '$snapshot',
-            properties: { $session_id: 'abcf-efg', $snapshot_data: { timestamp: 123 } },
+            properties: {
+                $session_id: sessionId,
+                $window_id: windowId,
+                $snapshot_data: { timestamp: 123, chunk_id: 'chunk_id', chunk_index: 'chunk_index' },
+            },
         } as any as PluginEvent,
         team.id,
         now,
         now,
         new UUIDT().toString()
     )
-    await delayUntilEventIngested(() => hub.db.fetchSessionRecordingEvents())
+    await delayUntilEventIngested(() => hub.db.fetchSessionRecordingEvents(sessionId))
 
     const events = await hub.db.fetchEvents()
     expect(events.length).toEqual(0)
 
-    const sessionRecordingEvents = await hub.db.fetchSessionRecordingEvents()
+    const sessionRecordingEvents = await hub.db.fetchSessionRecordingEvents(sessionId)
     expect(sessionRecordingEvents.length).toBe(1)
 
     const [event] = sessionRecordingEvents
-    expect(event.session_id).toEqual('abcf-efg')
+    expect(event.session_id).toEqual(sessionId)
     expect(event.distinct_id).toEqual('some-id')
-    expect(event.snapshot_data).toEqual({ timestamp: 123 })
+
+    const expectedFolderDate = castTimestampOrNow(now, TimestampFormat.DateOnly)
```

Comment:
Should this be a separate/new test rather than tacking new behavior onto a new one?

Reply:
I split the test (after first trying to move them into their own file) and they started not generating events... I'll leave this to a follow-up so I can concentrate on why that happened with less surrounding noise